Aspire Budgeting/Views/TabBarView/TabBarView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TabBarView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct TabBarView: View {
+  
+  private let cornerRadius: CGFloat = 16
+  private let height: CGFloat = 95
+  private let shadowRadius: CGFloat = 16
+  
+  private let prominentItemWidth: CGFloat = 70
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/TabBarView/TabBarView.swift#L15"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TabBarView.swift:15</code></td>
    </tr>
  </tbody>
</table>